patches/bug-reporting/matrix-react-sdk+3.95.0.patch

```diff
@@ -205,35 +192,53 @@ index 3c8241d..cdd9a83 100644
          const event = this.props.mxEvent;
          const sender = event.sender ? event.sender.name : event.getSender();
 diff --git a/node_modules/matrix-react-sdk/src/rageshake/submit-rageshake.ts b/node_modules/matrix-react-sdk/src/rageshake/submit-rageshake.ts
-index 5f2a399..3034aaa 100644
+index 5f2a399..abaaa24 100644
 --- a/node_modules/matrix-react-sdk/src/rageshake/submit-rageshake.ts
 +++ b/node_modules/matrix-react-sdk/src/rageshake/submit-rageshake.ts
-@@ -412,12 +412,13 @@ export async function submitFeedback(
-         version = await PlatformPeg.get()?.getAppVersion();
-     } catch (err) {} // PlatformPeg already logs this.
+@@ -102,7 +102,11 @@ async function collectBaseInformation(body: FormData, opts: IOpts): Promise<void
+     const touchInput = matchesMediaQuery("(pointer: coarse)");
  
+     body.append("text", opts.userText || "User did not supply any additional text.");
++    /* :TCHAP: rename app - for bugreport rageshakes
+     body.append("app", opts.customApp || "element-web");
++    */
++    body.append("app", "tchap-web");
++    // end :TCHAP:
+     body.append("version", version ?? "UNKNOWN");
+     body.append("user_agent", userAgent);
+     body.append("installed_pwa", installedPWA);
+@@ -129,6 +133,15 @@ async function collectClientInfo(client: MatrixClient, body: FormData): Promise<
+         await collectRecoveryInfo(client, cryptoApi, body);
+     }
+ 
++    // :TCHAP: add user email - for bugreport rageshakes
++    if(client) {
++        const result = await client.getThreePids();//it generates a API calls which is acceptable because feedbacks submit are not so frequent (unfortunately)
++        result.threepids.forEach(threepid => {
++            body.append(threepid.medium, threepid.address);
++        });
++    }
++    // end :TCHAP:
```

Comment:
This method seems to be used mutliple places in this file. Could we make it to a method ?